README.md

```diff
@@ -18,7 +18,7 @@ Project currently provides support for various package managers: `apt`, `yum`, `
 
 # Usage
 
-Currently there is no single program to run as it stands. Instead you can simply invoke the familiar `go test` on any of the examples in `examples` subdirectory. You can obviously create your own `_test.go` files for the stuff you want to test.
+Currently there is no single program to run as it stands. Instead the project provides API to build your own go binaries that can be used to test your infrastrucure - this way there is no need to have any runtime available on the tested infrastructure, just your binary. You can obviously create your own `_test.go` files for the stuff you want to test, but you will need to have go tool chain available on the tested server. Building binaries is recommended and preferrable option.
```

Comment:
well if you ship `_test.go` files you need go tool chain to be able to execute `go test`. Am I wrong ? The "preferable option" bit - this needs rewording I agree, but if you ship your "testing" binary you don't need go to be installed on the tested server.